node/lib/package/PackageInfo.ts

```diff
@@ -178,6 +186,39 @@ export class PackageTranslatorInfo {
     /**
      * Dictionary of properties passed to the onboarding module. For example, a property
      * may specify a filter for the kind of thing that is to be onboarded.
+     * This is optional and present only on actual devices.
      */
     public readonly onboardingProperties: { [propertyName: string]: string };
+
+
+    /**
+     * Onboarding information pertaining to the hub device, which will have the onboardign flow information. 
+     * For example, an 'input' flow type for acquiring username/client_id/client_secret
+     * This is optional and present only on hub devices.
+     */
+    public readonly onboardingFlow: OnboardingFlow[];
+}
+
+export class PackageOnboardingInfo
```

Comment:
Sure. I have added. I was mistaken, since I had moved onboarding flow to 'translator' structure. 

---
In reply to: [75431160](https://github.com/openT2T/opent2t/pull/22#discussion_r75431160) [](ancestors = 75431160)